projects/client-api-react/src/index.js

```diff
@@ -479,6 +490,24 @@ function Graphistry(props) {
         }
     }, [prevSub, gSub]);
 
+    useEffect(() => {
+        if (gObs && gSub && onUpdateObservableG) {
+            onUpdateObservableG({observable: gObs, subscription: gSub});
+            return () => onUpdateObservableG(null);
+        }
+    }, [gObs, onUpdateObservableG]);
+
+    useEffect(() => {
+        if (g && onSelectionUpdate) {
+            const sub = selectionUpdates(g).subscribe(
+                (v) => onSelectionUpdate(v),
+                (error) => console.error('client-api-react selectionUpdates subscription error', error)
```

Comment:
maybe we do node style here too, `onSelectionUpdate(err, val)` ?

Reply:
done.